react-components/src/API/_API.json

```diff
@@ -0,0 +1,76 @@
+{
+  "weatherAPI": [
+    {
+      "today": {
```

Comment:
this object demonstrates a good intention to write data with a good explanation...
But...
Any backend developer won't agree with you if you ask him to delivery the data in such format. "today", "tomorrow" and so on is a text that comes from a function... let's say
```js
function computeRelativeTime(ofDate: Date) {
  ...
  return relativeTime;
}
```

so, the response from the server shows a date or something similar... and you in the front-end side transform this relative date to text, perhaps in english or russian or ...